doozer/doozerlib/backend/rebaser.py

```diff
@@ -954,10 +956,27 @@ def _add_build_repos(self, dfp: DockerfileParser, metadata: ImageMetadata, dest_
             all_stages=True,
         )
 
+        # Parse the Dockerfile and find the occurrences of USERs
+        user_stanzas = []
+        for content in json.loads(dfp.json):
+            if "USER" in content.keys():
+                user_stanzas.append(f"USER {content['USER']}")
```

Comment:
Will this only find users in the final stage? We don't need or want to propagate USER from previous stages.

Reply:
`dfp.json` parses the entire Dockerfile